fairlearn/constants.py

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+demographic_parity = "demographic_parity"
+bounded_group_loss = "bounded_group_loss"
```

Comment:
why not DEMOGRAPHIC_PARITY and BOUNDED_GROUP_LOSS to match our convention for constants elsewhere?

Reply:
Yes, they should be (having just had a look at https://www.python.org/dev/peps/pep-0008/#constants)